planning/autoware_static_centerline_generator/package.xml

```diff
@@ -29,6 +29,7 @@
   <depend>geography_utils</depend>
   <depend>geometry_msgs</depend>
   <depend>global_parameter_loader</depend>
+  <depend>grid_map_ros</depend>
```

Comment:
Could you tell me the error output when if don't have grid_map_ros added into this dependency?
I have quickly searched the code with `grid_map` as a keyword in autoware_static_centerline_generator package, but I couldn't get any search results.

Reply:
I have some errors without `grid_map_ros`.

```
<command-line>: fatal error: grid_map_core/eigen_plugins/FunctorsPlugin.hpp: No such file or directory
compilation terminated.
gmake[2]: *** [CMakeFiles/main.dir/build.make:132: CMakeFiles/main.dir/src/utils.cpp.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....
<command-line>: fatal error: grid_map_core/eigen_plugins/FunctorsPlugin.hpp: No such file or directory
compilation terminated.
gmake[2]: *** [CMakeFiles/main.dir/build.make:118: CMakeFiles/main.dir/src/centerline_source/bag_ego_trajectory_based_centerline.cpp.o] Error 1
<command-line>: fatal error: grid_map_core/eigen_plugins/FunctorsPlugin.hpp: No such file or directory
compilation terminated.
gmake[2]: *** [CMakeFiles/main.dir/build.make:104: CMakeFiles/main.dir/src/centerline_source/optimization_trajectory_based_centerline.cpp.o] Error 1
<command-line>: fatal error: grid_map_core/eigen_plugins/FunctorsPlugin.hpp: No such file or directory
compilation terminated.
gmake[2]: *** [CMakeFiles/main.dir/build.make:90: CMakeFiles/main.dir/src/static_centerline_generator_node.cpp.o] Error 1
<command-line>: fatal error: grid_map_core/eigen_plugins/FunctorsPlugin.hpp: No such file or directory
compilation terminated.
gmake[2]: *** [CMakeFiles/main.dir/build.make:76: CMakeFiles/main.dir/src/main.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:625: CMakeFiles/main.dir/all] Error 2
```

This is caused by defining `EIGEN_DENSEBASE_PLUGIN` by `grid_map_ros`.
`EIGEN_DENSEBASE_PLUGIN` affects [Eigen behavior](https://eigen.tuxfamily.org/dox/TopicPreprocessorDirectives.html), and caues build error.
`grid_map_ros` is an indirect dependency of `static_centerline_generator`, but does not appear in include directory of `static_centerline_generator`.